charts/seed-operatingsystemconfig/original/templates/scripts/_health-monitor.sh

```diff
@@ -129,7 +129,7 @@
             fi
 
             # Check whether kubelet reports "PLEG not healthy" too often within the last 10 minutes and reboot VM if necessary.
-            if count_pleg_not_healthy="$(journalctl --since="$(date --date '-10min' "+%Y-%m-%d %T")" -u kubelet | grep "PLEG is not healthy" | wc -l)"; then
+            if count_pleg_not_healthy="$(journalctl --since="$(date --date '-10min' "+%Y-%m-%d %T")" -u kubelet | grep "PLEG is not healthy" | cut -f3 -d' '| cut -f2 -d':' | sort | uniq | wc -l)"; then
```

Comment:
Please add a comment with an example output of the Kubelet log and that this command only counts the an occurrence per minute.